spyder/app/tests/debug.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+import tqdm
+a = 1
```

Comment:
Please don't add this file and instead evaluate this two lines in the IPython console during the test. That would avoid adding `tqdm` as a dependency in Travis (which is not really needed).